pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -8,57 +11,78 @@
 
 class TestPaverBokChoyCmd(unittest.TestCase):
 
-    def setUp(self):
-        super(TestPaverBokChoyCmd, self).setUp()
-        self.request = BokChoyTestSuite('')
-
-    def _expected_command(self, expected_text_append, expected_default_store=None):
-        if expected_text_append:
-            expected_text_append = "/" + expected_text_append
+    def _expected_command(self, name, store=None):
+        """
+        Returns the command that is expected to be run for the given test spec
+        and store.
+        """
         shard = os.environ.get('SHARD')
         expected_statement = (
             "DEFAULT_STORE={default_store} "
             "SCREENSHOT_DIR='{repo_dir}/test_root/log{shard_str}' "
             "BOK_CHOY_HAR_DIR='{repo_dir}/test_root/log{shard_str}/hars' "
             "SELENIUM_DRIVER_LOG_DIR='{repo_dir}/test_root/log{shard_str}' "
-            "nosetests {repo_dir}/common/test/acceptance/tests{exp_text} "
+            "nosetests {repo_dir}/common/test/acceptance/{exp_text} "
             "--with-xunit "
             "--xunit-file={repo_dir}/reports/bok_choy{shard_str}/xunit.xml "
             "--verbosity=2 "
         ).format(
-            default_store=expected_default_store,
+            default_store=store,
             repo_dir=REPO_DIR,
-            exp_text=expected_text_append,
             shard_str='/shard_' + shard if shard else '',
+            exp_text=name,
         )
         return expected_statement.strip()
 
-    def test_default_bokchoy(self):
-        self.assertEqual(self.request.cmd.strip(), self._expected_command(''))
+    def test_default(self):
+        suite = BokChoyTestSuite('')
+        name = 'tests'
+        self.assertEqual(suite.cmd.strip(), self._expected_command(name=name))
```

Comment:
I don't know who put these in there (it could totally have been me), but it makes more sense to me if we took the strip() off both the _expected_command method and all these suite.cmd calls.